contracts/src/lib/PackedAccountIndex.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+/**
```

Comment:
you'll probably need to include this library in the anvil test deployments for the integration tests